doc/build-freebsd.md

```diff
@@ -36,7 +36,10 @@ export BDB_PREFIX="$PWD/db4"
 
 ## Building Bitcoin Core
 
-**Important**: Use `gmake` (the non-GNU `make` will exit with an error):
+**Important**: Use `gmake` (the non-GNU `make` will exit with an error).
+```shell
+setenv MAKE gmake
+```
```

Comment:
`setenv` is not portable between shells. Yes, it works on the default `csh` but does not work in others.

Given that `./configure MAKE=gmake` is an equivalent and works with any shell, I would suggest to add `MAKE=gmake` a few lines below where we show how to execute `./configure`.